example/https-example.js

```diff
@@ -1,22 +1,74 @@
-const https = require('https');
+/**
+ * Scenario:
+ * Generate requests towards specific domain
+ **/
+var trafficSimulator = require('../lib/main');
```

Comment:
This bug was because of keepaliveagent. I have fixed this error and will update the changes. Now the https request without throttling works. Is there any other test case I should check before pushing the updates?

Reply:
@iskitsas